qemu/tests/seabios_reboot_timeout.py

```diff
@@ -0,0 +1,50 @@
+import re
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    KVM seabios test:
+    1) Start guest with sga bios
+    2) Check if reboot-timeout option works as expected
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_output(session_obj):
+        """
+        Use the function to short the lines in the scripts
+        """
+        return session_obj.get_stripped_output()
+
+    def reboot_timeout_check():
+        """
+        reboot-timeout check
+        """
+        return re.search(pattern, get_output(seabios_session), re.S)
+
+    error_context.context("Start guest with sga bios", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    timeout = float(params.get("login_timeout", 360))
+    seabios_session = vm.logsessions['seabios']
+    rb_timeout = int(params["boot_reboot_timeout"])/1000
+    if rb_timeout < 0:
+        raise NotImplementedError()
```

Comment:
Is there any chance boot_reboot_timeout set to < 0?
If not, should it be better to raise error because it's an invalid argument.

Reply:
Yes, there is a product bug tracking 'boot_reboot_timeout < 0', will update this part after the bug fixed. Thanks!